Description: Removes the use of the de.unkrig.commons.nullanalysis annotations
 which aren't provided with the upstream sources.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/janino-dist/build.xml
+++ b/janino-dist/build.xml
@@ -88,21 +88,6 @@
 		/>
 
 		<!-- Build 'build/commons-compiler-jdk.jar'. -->
-		<delete dir="build/bin-de.unkrig.commons.nullanalysis"/>
-		<mkdir dir="build/bin-de.unkrig.commons.nullanalysis"/>
-		<javac
-			fork="yes"
-			source="1.6"
-			target="1.6"
-			srcdir="../de.unkrig.commons.nullanalysis/src"
-			destdir="build/bin-de.unkrig.commons.nullanalysis"
-			debug="true"
-			debuglevel="lines,vars,source"
-			includeantruntime="false"
-		>
-			<compilerarg value="-Xlint:none"/>
-			<compilerarg value="-Xlint:deprecation"/>
-		</javac>
 		<delete dir="build/bin-commons-compiler-jdk"/>
 		<mkdir dir="build/bin-commons-compiler-jdk"/>
 		<javac
@@ -115,6 +100,7 @@
 			debug="true"
 			debuglevel="lines,vars,source"
 			includeantruntime="false"
+			excludes="**/org/codehaus/commons/compiler/jdk/package-info.java"
 		>
 			<compilerarg value="-Xlint:none"/>
 			<compilerarg value="-Xlint:deprecation"/>
@@ -128,7 +114,6 @@
 				<attribute name="Implementation-URL"     value="http://janino.net"/>
 			</manifest>
 			<fileset dir="build/bin-commons-compiler-jdk"/>
-			<fileset dir="build/bin-de.unkrig.commons.nullanalysis"/>
 		</jar>
 		<signjar
 			keystore="dummy-keystore"
